### PR TITLE
Add profile for azure stack

### PR DESCRIPTION
**Pull request by ChenTanyi**

### CODE REVIEW

File: profiles/2020-09-01-hybrid/azure/src/main/java/com/microsoft/azure/management/profile_2019_03_01_hybrid/CustomCipherSuites.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+package com.microsoft.azure.management.profile_2019_03_01_hybrid;
+
+import okhttp3.CipherSuite;
+import org.apache.commons.lang3.ArrayUtils;
+
+/**
+ * Collects cipher suites in OkHttp for needed.
+ */
+public class CustomCipherSuites {
```
Comment (weidongxu-microsoft) [inline profiles/2020-09-01-hybrid/azure/src/main/java/com/microsoft/azure/management/profile_2019_03_01_hybrid/CustomCipherSuites.java:15]: Just curious, any background on this?

**Review by weidongxu-microsoft — COMMENTED** [review]
Please involve azure stack team as reviewer.

File: profiles/2020-09-01-hybrid/azure/src/main/java/com/microsoft/azure/management/profile_2019_03_01_hybrid/CustomCipherSuites.java (hunk 1 above)
Comment (ChenTanyi) [reply profiles/2020-09-01-hybrid/azure/src/main/java/com/microsoft/azure/management/profile_2019_03_01_hybrid/CustomCipherSuites.java:15]: Yes, I am sending email for them.

For Azure Stack, the default `CipherSuites` in `OkHttp` cannot send successful request. (Due to their security concern)

**Comment by JonathanGiles** [comment]
@ChenTanyi Any update on this or should the PR be closed?

**Pull request closed without merging**